lms/envs/common.py

```diff
@@ -51,11 +51,6 @@
 
 PLATFORM_FACEBOOK_ACCOUNT = "http://www.facebook.com/YourPlatformFacebookAccount"
 PLATFORM_TWITTER_ACCOUNT = "@YourPlatformTwitterAccount"
-PLATFORM_TWITTER_URL = "https://twitter.com/YourPlatformTwitterAccount"
```

Comment:
ah my mistake, I found PLATFORM_TWITTER_ACCOUNT, not URL. You should be all
set!

On Tue, Mar 17, 2015 at 1:37 PM, Will Daly <notifications@github.com> wrote:

> In lms/envs/common.py
> <https://github.com/edx/edx-platform/pull/7383#discussion_r26595974>:
>
> > @@ -51,11 +51,6 @@
> >
> >  PLATFORM_FACEBOOK_ACCOUNT = "http://www.facebook.com/YourPlatformFacebookAccount"
> >  PLATFORM_TWITTER_ACCOUNT = "@YourPlatformTwitterAccount"
> > -PLATFORM_TWITTER_URL = "https://twitter.com/YourPlatformTwitterAccount"
>
> Hmm, I can't seem to find them when I grep the code base. (I'm finding the
> _ACCOUNT settings, which aren't deleted, but not the _URL settings).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/7383/files#r26595974>.
>